.github/dependabot.yml

```diff
@@ -6,8 +6,32 @@ updates:
       interval: "weekly"
```

Comment:
Can you change the interval to run on the first Thursday of every month, we don't need to check for updates weekly and any vulnerabilities will get picked up separately.